app/scripts/views/app.coffee

```diff
@@ -569,10 +564,22 @@ define [
       # TODO @jrwdunham: use the already-in-place @alertDialog for this
       # TODO @jrwdunham @cesine: figure out how i18n/localization works in
       # FieldDB and begin implementing something similar in Dative.
-      console.log ["TODO show some visual thing here using the app view using",
+      console.log ["Pretending the user said Yes. This is dangerous. TODO show some visual confirm dialog here using the app view using",
```

Comment:
This weird string syntax was before I understood how CoffeeScript multiline strings work. You can just add linebreaks and proper indentation in the middle of the string and it will all work out, i.e., remove the joined array.